Frontend/VIAProMa/Assets/AnchoringMenu.cs

```diff
@@ -0,0 +1,250 @@
+    /// When activated only the 3D model is moved upon manipulation, otherwise the entire anchor parent is transformed
+    /// </summary>
+    public void OnCheckboxMoveAnchorClicked()
+    {
+        if (anchorManager.moveAnchorAloneEnabled)
+        {
+            DisableMoveAnchorAlone();
+        }
+        else
+        {
+            EnableMoveAnchorAlone();
+        }       
+    }
+
+    /// <summary>
+    /// Called when the user locks or unlocks the anchor
+    /// The status of the object manipulator component is updated together with the button icon and corresponding status text
+    /// </summary>
+    public void OnLockButtonClicked()
+    {
+        if (!anchorManager.anchorLocked)
+        {
+            EnableAnchorLock();
+
+        }
+        else
+        {
+            DisableAnchorLock();
+        }
+    }
+
+    /// <summary>
+    /// Enables the anchor lock and updates the corresponding components
+    /// </summary>
+    public void EnableAnchorLock()
+    {
+        anchorManager.anchorLocked = true;
+        anchorObject.GetComponent<ObjectManipulator>().enabled = false;
+        lockedIconSpriteRenderer.sprite = lockedIcon;
+        lockButtonText.text = "Anchor is locked";
+        Debug.Log("The anchor is now locked.");
+    }
+
+    /// <summary>
+    /// Disables the anchor lock and updates the corresponding components
+    /// </summary>
+    public void DisableAnchorLock()
+    {
+        anchorManager.anchorLocked = false;
+        anchorObject.GetComponent<ObjectManipulator>().enabled = true;
+        lockedIconSpriteRenderer.sprite = unlockedIcon;
+        lockButtonText.text = "Anchor is unlocked";
+        Debug.Log("The anchor is now unlocked.");
+    }
+
+    /// <summary>
+    /// Disables anchoring and all corresponding UI components.
+    /// </summary>
+    public void DisableAnchoring()
+    {
+        anchorManager.anchoringEnabled = false;
+        useAnchoringCheckbox.IsToggled = false;
+        anchorObject.SetActive(false);
+        extendedSettings.SetActive(false);
+        Debug.Log("Anchoring has been deactivated.");
+    }
+
+    /// <summary>
+    /// Enables anchoring and all corresponding UI components.
+    /// </summary>
+    public void EnableAnchoring()
+    {
+        anchorManager.anchoringEnabled = true;
+        anchorObject.SetActive(true);
+        extendedSettings.SetActive(true);
+        Debug.Log("Anchoring has been activated.");
+    }
+
+    /// <summary>
+    /// Enables the functionality to move the anchor independently of the scene
+    /// </summary>
+    public void EnableMoveAnchorAlone()
+    {
+        anchorManager.moveAnchorAloneEnabled = true;
+        moveAnchorAloneCheckbox.IsToggled = true;
+        anchorObject.GetComponent<ObjectManipulator>().HostTransform = anchorObject.transform;
+        Debug.Log("Individual moving mode enabled.");
+    }
+
+    /// Disables the functionality to move the anchor independently of the scene
+    /// </summary>
```

Comment:
There is an opening tag for the summary XML node missing.

```suggestion
    /// <summary>
    /// Disables the functionality to move the anchor independently of the scene
    /// </summary>
```